dynamic_listings/python/default/collisiondetection/line_line.py

```diff
@@ -13,8 +11,6 @@ def lineLineCollision(
     uB: float = ((x2 - x1) * (y1 - y3) - (y2 - y1) * (x1 - x3)) / den
 
     # Let's see if uA and uB tell us the lines are colliding
-    if (0 <= uA <= 1) and (0 <= uB <= 1):
-        return True
-
+    is_colliding = 0 <= uA <= 1 and 0 <= uB <= 1
```

Comment:
I would, personally, wrap the right-hand side in parentheses, since it may look a bit unclear.